src/isIsogram.test.js

```diff
@@ -6,4 +6,35 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
```

Comment:
Same here - I would add two more basic checks according to the checklist:
https://mate-academy.github.io/qa-program/checklists/unit_tests.html#first-two-tests-should-validate-that-function-is-declared-and-data-type-the-function-returns